lib/store_attribute/active_record/type/typed_store.rb

```diff
@@ -84,7 +98,16 @@ def type_for(key)
         accessor_types.fetch(key.to_s)
       end
 
-      attr_reader :accessor_types, :store_accessor
+      def has_default?(key)
+        defaults.key?(key.to_s)
+      end
+
+      def get_default(key)
+        value = defaults.fetch(key.to_s)
```

Comment:
We always use it with string input

```suggestion
        value = defaults.fetch(key)
```